staker-rewards/rewards.js

```diff
@@ -71,21 +75,39 @@ function validateIntervalTimestamps(interval) {
     )
   }
 
-  console.log(clc.green(`Interval start: ${startDate.toISOString()}`))
-  console.log(clc.green(`Interval end: ${endDate.toISOString()}`))
+  console.log(clc.green(`Interval start timestamp: ${startDate.toISOString()}`))
+  console.log(clc.green(`Interval end timestamp: ${endDate.toISOString()}`))
+}
+
+async function determineIntervalBlockspan(context, interval) {
+  const blockByDate = new BlockByDate(context.web3)
+
+  interval.startBlock = (await blockByDate.getDate(interval.start * 1000)).block
+  interval.endBlock = (await blockByDate.getDate(interval.end * 1000)).block
+
+  console.log(clc.green(`Interval start block: ${interval.startBlock}`))
+  console.log(clc.green(`Interval end block: ${interval.endBlock}`))
 }
 
 async function calculateOperatorsRewards(context, interval) {
   const { cache } = context
   const slaCalculator = await SLACalculator.initialize(context, interval)
+  const assetsCalculator = await AssetsCalculator.initialize(context, interval)
 
   const operatorsRewards = []
 
   for (const operator of getOperators(cache)) {
+    console.log(`Calculating summary for operator ${operator}`)
```

Comment:
I'd rather add a log in `calculateOperatorAssets` saying `Calculating KEEP and ETH under management for operator ${operator}` so that we are precise in logs about what's currently happening. All the other logs that are previously printed inform very precisely what step we are at.